generate.sh

```diff
@@ -24,7 +24,10 @@ commands=""
 while read name description
 do
   name="$(echo $name | sed -e 's/:/\\:/g')"
-  commands="${commands}\n\t\"$name\":\"$description\""
+  description="$(echo $description | cut -d . -f 1)"
+  if [[ $name == force\\:* ]]; then
+    commands="${commands}\n\t\"${name//\"/\\\"}\":\"${description//\"/\\\"}\""
+  fi
```

Comment:
* Line 27 and 28 ensure only `force` commands are generated in the `_1st_arguments` variable.
* Line 29 escapes double quotes that if unescaped cause autocompletion script to fail.

![image](https://user-images.githubusercontent.com/4142577/60752945-b8bd5180-9f91-11e9-8486-6b043fb39862.png)